cmd/node/main.go

```diff
@@ -664,6 +665,15 @@ func startNode(ctx *cli.Context, log logger.Logger, version string) error {
 		return err
 	}
 
+	rounder, err := round.NewRound(
```

Comment:
Please pass this rounder to NewProcessComponentsFactoryArgs right below in line 906, and remove its second creation from func ProcessComponentsFactory(args *processComponentsFactoryArgs) (*Process, error), to have only one instance of rounder in the whole project

Reply:
done